### ISSUE TITLE
weird error in ionic app

### ISSUE THREAD

**Issue by ribrdb** [firebase/firebaseui-web#326]
I'm just trying to do a basic login in an ionic app. I'm using a redirect since it will eventually use cordova. Doesn't matter which provider I choose, after the login redirect I get a strange exception:
```
cancel
    at new gt (https://www.gstatic.com/firebasejs/4.10.0/firebase.js:13:9299)
    at Z.<anonymous> (https://www.gstatic.com/firebasejs/4.10.0/firebase.js:13:81605)
    at $ (https://www.gstatic.com/firebasejs/4.10.0/firebase.js:13:6669)
    at t.invoke (http://localhost:8100/build/polyfills.js:3:14976)
    at r.run (http://localhost:8100/build/polyfills.js:3:10143)
    at http://localhost:8100/build/polyfills.js:3:20242
    at t.invokeTask (http://localhost:8100/build/polyfills.js:3:15660)
    at r.runTask (http://localhost:8100/build/polyfills.js:3:10834)
    at o (http://localhost:8100/build/polyfills.js:3:7894)
```

This is completely unhelpful, I've got no idea how to debug this.

**Comment by wti806** [comment]
Are you using FirebaseUI or Firebase JS SDK?

**Comment by ribrdb** [comment]
I'm using FirebaseUI, but it doesn't appear in this stack trace.

**Comment by wti806** [comment]
Can you provide your signInSuccess callback and the firebaseUI version you are using ? And also do you call ui.reset() after signed in?

**Comment by rb010** [comment]
Same issue!
It only works when I change signInSuccess to true, but then I get the error informing that there is no redirect url.
firebaseui: 2.6.1

**Comment by wti806** [comment]
Looks like the issue is same as #310. It happens when you return false in signInSuccess callback or you call reset after signing in. The bug was introduced in 2.6.0 and 2.6.1. We've already have a fix for it and will release shortly in 2.6.2. Will keep you guys posted. Thanks for reporting the issue!

**Comment by wti806** [comment]
Fixed in release 2.6.2. https://github.com/firebase/firebaseui-web/releases
